cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/AppFabricServerTest.java

```diff
@@ -41,13 +63,14 @@
 
   @BeforeClass
   public static void before() throws Exception {
-    Injector injector = AppFabricTestHelper.getInjector();
-    server = injector.getInstance(AppFabricServer.class);
-    discoveryServiceClient = injector.getInstance(DiscoveryServiceClient.class);
+
   }
 
   @Test
   public void startStopServer() throws Exception {
+    Injector injector = AppFabricTestHelper.getInjector();
+    server = injector.getInstance(AppFabricServer.class);
```

Comment:
``server`` and ``discoveryServiceClient`` don't need to be class members now, right?